wolfcrypt/test/test.c

```diff
@@ -24647,12 +24606,19 @@ WOLFSSL_TEST_SUBROUTINE int ecc_encrypt_test(void)
     wc_ecc_ctx_free(srvCtx);
     wc_ecc_ctx_free(cliCtx);
     /* let's verify message exchange works, A is client, B is server */
-    cliCtx = wc_ecc_ctx_new(REQ_RESP_CLIENT, &rng);
-    srvCtx = wc_ecc_ctx_new(REQ_RESP_SERVER, &rng);
+    cliCtx = wc_ecc_ctx_new(REQ_RESP_CLIENT, rng);
+    srvCtx = wc_ecc_ctx_new(REQ_RESP_SERVER, rng);
     if (cliCtx == NULL || srvCtx == NULL) {
         ret = -10416; goto done;
     }
 
+    ret = wc_ecc_ctx_set_algo(cliCtx, encAlgo, kdfAlgo, macAlgo);
+    if (ret != 0)
+        goto done;
+    ret = wc_ecc_ctx_set_algo(srvCtx, encAlgo, kdfAlgo, macAlgo);
+    if (ret != 0)
+        goto done;
+
```

Comment:
Below at 24643 and 24646 there is an existing issue.. `wolfSSL MSGE` is 12 characters, not 11...